packages/tools/devtools/devtools-view/src/TelemetryUtils.ts

```diff
@@ -42,9 +44,17 @@ export function useLogger(): ITelemetryLoggerExt | undefined {
  * in the brower's devtools panel, selecting "Inspect", and switching to the Console tab.
  */
 export class ConsoleVerboseLogger implements ITelemetryBaseLogger {
-	public constructor(private readonly baseLogger?: ITelemetryBaseLogger) {}
+	private readonly sessionID?: string;
+	private readonly continuityID?: string;
+	public constructor(private readonly baseLogger?: ITelemetryBaseLogger) {
+		this.sessionID = uuidv4();
+		this.continuityID = getOrCreateBrowserId();
+	}
 
 	public send(event: ITelemetryBaseEvent): void {
+		event.sessionID = this.sessionID;
+		event.continuityID = this.continuityID;
+
```

Comment:
I don't think we need this in the console logger, where the users already have some "continuity" to see that all messages written to console are related to each other.

Reply:
Should these identifiers be removed entirely from the console logger?

Reply:
Yeah, in general I don't think we need any changes in the devtools-view package to implement this.